pkgs/by-name/ax/axmldec/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, cmake, doxygen, boost, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "axmldec";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    repo = "axmldec";
+    owner = "ytsutano";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-LFDZZbRDa8mQmglgS4DA/OqXp0HJZ2uqg1hbStdgvUw=";
+  };
+
+  nativeBuildInputs = [ cmake doxygen ];
+  buildInputs = [ boost zlib ];
+
+  meta = with lib; {
+    mainProgram = "axmldec";
+    description = "Stand-alone binary AndroidManifest.xml decoder";
+    longDescription = ''
+      This tool accepts either a binary or a text XML file and prints the
+      decoded XML to the standard output or a file. It also allows you to
+      extract the decoded AndroidManifest.xml directly from an APK file.
+    '';
+    homepage = "https://github.com/ytsutano/axmldec";
+    license = licenses.isc;
+    maintainers = with maintainers; [ franciscod ];
+    platforms = platforms.unix;
```

Comment:
```suggestion
    platforms = platforms.unix ++ platforms.windows;
```

Reply:
wait what there's nix for windows?

Reply:
No, not really. But we keep a long list of architectures and try to keep the packages' list of supported ones equal to what upstream specifies. Have a look at https://github.com/NixOS/nixpkgs/blob/master/lib/systems/doubles.nix if you're interested